examples/resources/gitlab_group_project_file_template/resource.tf

```
@@ -0,0 +1,16 @@
+resource "gitlab_group" "foo" {
+  name        = "group"
+  path        = "group"
+  description = "An example group"
+}
+
+resource "gitlab_project" "bar" {
+  name             = "template project"
+  description      = "contains file templates"
+  visibility_level = "public"
```

It's missing the very crucial `namespace_id` here, right?

```suggestion
  visibility_level = "public"
  
  namespace_id = gitlab_group.foo.id
```

---

well now I'm curious how all the tests pass without that... I'll do some digging here.

---

It's only missing in the example :)

---

Whew. little bit of panic there for a second 😂